maker/src/bitmex.rs

```diff
@@ -0,0 +1,161 @@
+use anyhow::Result;
+use futures::TryStreamExt;
+use rust_decimal::Decimal;
+use std::str::FromStr;
+use time::OffsetDateTime;
+use tokio::sync::watch;
+use tokio::task::JoinHandle;
+
+pub const QUOTE_INTERVAL_MINUTES: i64 = 1;
+
+pub fn subscribe() -> Result<(JoinHandle<()>, watch::Receiver<Option<Quote>>)> {
+    let (quote_sender, quote_receiver) = watch::channel(None);
+
+    let handle = tokio::spawn(async move {
+        let mut stream = bitmex_stream::subscribe(
+            ["instrument:XBTUSD".to_string()],
+            bitmex_stream::Network::Testnet,
+        );
+
+        while let Some(text) = stream.try_next().await.expect("message from bitmex") {
+            tracing::trace!(%text, "Received message from bitmex");
+            let quote = Quote::from_str(&text).expect("quote from bitmex");
+            if let Some(quote) = quote {
+                tracing::debug!(
+                    "Received quote update for {:?}, bid: {}, ask: {}, index: {}",
+                    quote.symbol,
+                    quote.bid,
+                    quote.ask,
+                    quote.index,
```

Comment:
You could also log the timestamp we get from BitMEX.

Reply:
fixed f533eb99bafbbd8f6381ff23a2b644e915df0c9d